tests/core/test_subset.py

```diff
@@ -1056,7 +1056,6 @@ def test_rectilinear(self):
             inds = subset.shape_bbox_indexer(ds, gpd.GeoDataFrame(geometry=[pb]))
             assert pb.within(subset.grid_exterior_polygon(ds.isel(inds)))
 
-    @pytest.mark.skip
```

Comment:
You need `@pytest.mark.skipif(xesmf is None, reason="xESMF is needed for  this test.")`

Reply:
So you suggest I just update the results with the most recent values ?

Reply:
I think so yeah. You could verify locally that the tests pass with xESMF 0.6.1?

Reply:
Did it with 0.6.0. I still had issues with 0.6.1.